tools/arm_pack_manager/__init__.py

```diff
@@ -339,7 +343,8 @@ def index(self) :
                 with open(join(save_data_path('arm-pack-manager'), "index.json")) as i :
                     self._index = load(i)
             except IOError :
-                self.generate_index()
+                with open(LocalPackIndex) as i :
```

Comment:
Good catch - seems like the update functionality should be removed and this data only delivered (looked for) when tested and part of a release.